pkg/kubelet/config/file.go

```diff
@@ -44,6 +45,9 @@ type sourceFile struct {
 }
 
 func NewSourceFile(path string, nodeName types.NodeName, period time.Duration, updates chan<- interface{}) {
+	// "golang.org/x/exp/inotify" requires a path without tailing "/"
+	path = strings.TrimRight(path, "/")
```

Comment:
os.PathSeparator is "/":
https://golang.org/pkg/os/#PathSeparator
And I think now kubelet binary only support linux?